m-queue.js

```diff
@@ -201,16 +234,10 @@ module.exports = function(RED) {
                     }
                 } else {
                     qIndex = findQueue(qSelect,qArray)
-                    }        
                     if (qIndex < 0) {   // create new queue
-                        qArray.push({
```

Comment:
And the last one. If your default size is zero, the queue is created with a message in it.